package.json

```diff
@@ -44,6 +44,7 @@
     "babel-eslint": "^8.2.6",
     "babel-loader": "^7.1.5",
     "babel-preset-env": "^1.6.1",
+    "browser-sync": "^2.24.7",
```

Comment:
Don't forget to remove gulp-livereload if we're not using it anymore. Also update the Readme file with a link to the new package being used